lib/internal/Magento/Framework/Exception/AggregateExceptionInterface.php

```diff
@@ -0,0 +1,21 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\Framework\Exception;
+
+/**
+ * Exception with possibility to set several error messages
+ *
```

Comment:
AggregateExceptionInterface Extension Point introduced to support Multiple Errors returned as a result of Validation not mandating to inherit from  AbstractAggregateException class